.github/workflows/operator-regression.yml

```diff
@@ -23,6 +27,33 @@ jobs:
         run:
            bash test/operator/kind-with-registry.sh
 
+      - name: Start Elasticsearch
+        run: |
+           docker network create elastic
+           #Local elasticsearch without SSL on http://localhost:9200
+           docker run --name es01 --net elastic -p 9200:9200 -m 1GB -e xpack.security.enabled=false -e xpack.security.enrollment.enabled=false -e discovery.type=single-node docker.elastic.co/elasticsearch/elasticsearch:8.16.1 1>es.log 2>&1 &
+           bash test/operator/utilities/wait_for_es_http_ready.bash http://localhost:9200
+
+      - name: Start Collector
```

Comment:
Could you provide more information about why this Collector instance is needed?  Why not using the Operator's endpoint?

Reply:
The flow of trace data is app->app agent->collector->ES, so it needs a collector. The operator endpoint is for mutating the k8s definitions so that's entirely separate, that's not involved in the flow of traces

Reply:
I might be missing something, but if you are using the `values.yaml` file we ship to deploy the Operator, it will deploy a collector in each K8s node with an open OTLP endpoint for traces. Those collectors will have the same processing components that the one you are launching manually.

Reply:
If the data already has the right shape, then your collector here could just receive data and forward it to ES, if you really want to have it. It doesn't need to use this full config.

Though if it was up to me, I'd just ES itself in the K8s cluster and forward the 9200 port.

Reply:
The collector does some transforms on the incoming data from the application agents, so I think it needs to be correctly configured for handling incoming trace data.

Reply:
> The collector does some transforms on the incoming data from the application agents, so I think it needs to be correctly configured for handling incoming trace data.

Correct, I would use the collector deployed by the Operator as it is already configured to handle/transform any OTLP traces.

> Though if it was up to me, I'd just ES itself in the K8s cluster and forward the 9200 port.

That makes sense to me as well, we will just need to maintain one type of deploying infrastructure (K8s/kind).